test/unittests/test_create_project.py

```diff
@@ -72,5 +72,27 @@ def test_circular_references_in_onto(self) -> None:
         self.assertListEqual(sorted(errors), sorted(expected_errors))
 
 
+    def test_check_for_undefined_cardinalities(self) -> None:
```

Comment:
As far as I understand, you're only testing the negative case. It would be good to also test the positive case where the method returns `True`.

Reply:
thanks for the hint, I have now added such a case